src/LogsSQL.php

```diff
@@ -63,6 +63,42 @@ public function migrationWasExecuted(LogStrategy $strategy, string $connectionSt
         return false;
     }
 
+    public function seedWasExecuted(LogStrategy $strategy, string $connectionString, string $seed): bool
+    {
+        if (!$strategy instanceof LogStrategySQL) {
+            return false;
+        }
+
+        $db = new PDO(
+            $strategy->getConnectionString(),
+            $strategy->getUser(),
+            $strategy->getPassword(),
+            [PDO::ATTR_ERRMODE => PDO::ERRMODE_EXCEPTION]
+        );
+
+        $query = "SELECT * FROM `{$strategy->getTable()}`";
```

Comment:
```suggestion
        $query = "SELECT `event` FROM `{$strategy->getTable()}`";
```

Looking from the code it seemed that only `event` was used. (but best to double check). It is always better to specify the required columns here, both for readability and to reduce bandwidth

Reply:
So this was a copy pasta, but indeed, for the seeds we're only checking for the event, so, you ask wij draaien eh.